Utils/Functions/Net/Download/script.js

```diff
@@ -119,19 +119,20 @@ module.default = class Downloader {
             this._message = tr("Please wait while {0} is downloaded...", this._fetchFileNameFromUrl(this._url));
         }
 
+        let progressBar;
         if (this._wizard) {
-            var progressBar = this._wizard.progressBar(this._message);
```

Comment:
The previous call led to `progressBar` being available only inside the if block but not afterwards